docs/handbook/01_intro.md

```diff
@@ -0,0 +1,19 @@
+Reaching towards the hardware level, we also consider tools for _quantum error correction_ {cite:p}`berentDecodingQuantumColor2023,berentSoftwareToolsDecoding2023,strikisQuantumLowdensityParitycheck2023` and _physical design_ {cite:p}`kunasaikaranFrameworkDesignRealization2024`.
+In all these tools, we try to utilize _data structures and core methods_ facilitating the efficient handling of quantum computations.
+The proposed solutions demonstrate how utilizing design automation expertise can lead to improved efficiency, scalability, and reliability.
+They illustrate the immense benefits of leveraging expertise in classical circuit and system design rather than starting from scratch.
+All tools developed as part of the MQT are made available as open-source packages on [github.com/cda-tum](https://github.com/cda-tum/).
+
+In the following, we briefly summarize the main tools (covering classical simulation, compilation, and verification of quantum circuits as well as benchmarking).
```

Comment:
Perhaps this sentence is a good place to reference back to the `core methods`:
"In the following, we briefly summarize the main core tools ... ", or similar?

Reply:
Slighlty changed the wording in https://github.com/cda-tum/mqt/pull/10/commits/24174d3a4d15e216f3186092fda99cb3cf028738 👍🏼